tests/test_data/output.txt

```diff
@@ -8,28 +8,30 @@ import something  # with comment
 from a import b
 from a.b import c, d
 from other import (  # noqa
-    # lots of happy things below
-    and_rainbows,
-    something,  # something comment
-    something_else,
-)
-from other.package.subpackage.module.submodule import (  # noqa
+                   # lots of happy things below
+                   and_rainbows,
+                   something,  # something comment
+                   something_else,
+                   )
+from other.package.subpackage.module.submodule import (
     CONSTANT,
     Klass,
+    Supercalifragilisticexpialidocious,
     bar,
     foo,
     rainbows,
-)
+    )
 from package.subpackage.module.submodule import (
-    CONSTANT,
-    Klass,
-    bar,
-    foo,
-    rainbows,
-)
+                                                 CONSTANT,
```

Comment:
not sure if this would be expected output. I would expect something like:

```python
from package.subpackage.module.submodule import (CONSTANT,
                                                 Klass,
                                                 bar,
                                                 foo,
                                                 rainbows)
```